content/developer/api/version-8/Requirements.adoc

```diff
@@ -21,3 +21,9 @@ Please ensure that you have the following:
 * OpenSSL PHP Extension installed and configured
 * The SuiteCRM instance must be configured to use HTTPS/SSL
 * You need to have PHP version 5.5.9, or 7.0 and above
+
+To be able to successfully connect to the endpoints, you need to complete the steps under 
+
+https://docs.suitecrm.com/developer/api/version-8/json-api/#_before_you_start_calling_endpoints 
```

Comment:
Please make internal link relative.

I _think_ the correct way to do it will be this 

`To be able to successfully connect to the endpoints, you need to complete the steps under section link:../json-api/#_before_you_start_calling_endpoints[Before you start calling endpoints]`

... but please test it, it might need one more `..` or one less, I never get it right at first :-)

Reply:
I am not familliar with AsciiDOC, so didn't think about that at all. Also didn't find a good way to test this within GitHub, and I don't have an doc devenv running, but it may be right now (.(https://asciidoctor.org/docs/user-manual/#navigating-between-source-files, and JSON-API is on the same level as requirements, not sure if capitalization was necessary but it shouldn't hurt).
Sorry for the mess!

Reply:
I am thankful when people try to make PR's to Documentation, even when they don't get things right at first. There's a learning curve here, that's perfectly normal.

If you don't have a local environment, you can still test using the automatic Netlify "preview deploys". On the bottom of this PR, click `Show all checks` and then `Details`, and you are taken either to a build error page (if the build fails), or to a browseable preview of the site.

You will see that your link is actually missing a `..`. This is because Hugo (our content engine) uses a separate directory for each page (just to make the URL's prettier), so in the rendered site you actually have to go up one level to get to a page that was in the same level in the source files.

Thanks for contributing, I really appreciate the help, especially on API pages!

Reply:
Another thing - maybe we don't need the separate paragraphs, maybe this could read just like a sentence?



To be able to successfully connect to the endpoints, you need to complete the steps under section [Before you start calling endpoints](http://some.url) in addition to what is mentioned here.